staging/us-east-2/modules/main.tf

```diff
@@ -284,16 +284,153 @@ resource "aws_launch_template" "lt" {
 ####### Create Auto Scaling Group
 ##########################################################################
 
-resource "aws_autoscaling_group" "staging_asg" {
+resource "aws_autoscaling_group" "asg" {
   name = "${var.env}-asg"
-#   target_group_arns   = [aws_lb_target_group.staging_tg.arn]
+  target_group_arns   = [aws_lb_target_group.tg.arn]
   vpc_zone_identifier = [ for subnet in aws_subnet.private_subnets : subnet.id]
   desired_capacity    = 2
   max_size            = 4
   min_size            = 1
+  health_check_grace_period = 10
+  health_check_type         = "ELB"
 
   launch_template {
     id      = aws_launch_template.lt.id
     version = aws_launch_template.lt.latest_version
   }
-}
\ No newline at end of file
+}
+
+##########################################################################
+####### Create Target Group
+##########################################################################
+
+resource "aws_lb_target_group" "tg" {
+  name     = "${var.env}-80-tg"
+  port     = 80
```

Comment:
Not a blocker but you could replace the hardcoded **80** with your **var.lb_port_http**

Reply:
I will update this and make sure I have no other values that are hardcoded in the module and comit the change before merging. Thanks for that cath.

Reply:
Updated this and any other place where values were hardcoded to use variabes. It looks better now.